docs/.vuepress/config.js

```diff
@@ -26,7 +26,20 @@ const sidebar = {
       children: [
         ['/developer-docs/latest/development/backend-customization.md', 'Backend customization'],
         ['/developer-docs/latest/development/admin-customization.md', 'Admin panel customization'],
-        ['/developer-docs/latest/development/plugin-customization.md', 'Strapi plugins'],
+        {
+          title: 'Strapi plugins',
+          path: '/developer-docs/latest/development/plugin-customization.html',
+          collapsable: true,
+          children: [
```

Comment:
Unrelated to this PR specifically but I do like this, we should probably do it for the deployment also

Reply:
Thanks for your feedback, glad you like it :-)
Will do this in another dedicated PR.